src/report/utility-reports/hello-world.scm

```diff
@@ -96,7 +96,17 @@
      (gnc:make-string-option
       (N_ "Hello, World!") (N_ "String Option")
       "c" (N_ "This is a string option.") (N_ "Hello, World")))
-    
+
+    ;; This is an eval option. Users can type a scheme expression.
+    ;; The default value is "(+ 2 3)". The scheme expression is 
+    ;; evaluated left to right, and the last value returned is
+    ;; to be printed into the report. This will be useful to 
+    ;; learn hacking scheme in gnucash.
+    (add-option
+     (gnc:make-string-option
+      (N_ "Hello, World!") (N_ "Eval Option")
+      "cc" (N_ "This is an eval option.") (N_ "(+ 2 3)")))
```

Comment:
As the default value is not a string you expect to translate there is no need to wrap it in (N_  ...). In fact it's even better not to because as written "(+ 2 3)" will be added to the list of strings to translate...